### ISSUE TITLE
Exception when using with git-worktree

### ISSUE THREAD

**Issue by Borph** [gitextensions/gitextensions#2960]
When creating another workdir with

`git worktree add <path-to-new-dir> <branch>`

I can successfully work in that folder with git commands on the console. It is a new working directory from the same repository (https://git-scm.com/docs/git-worktree).

However, when opening the folder with Git-Extension (Win7-64bit), I get an exception and application closes:

```
System.ArgumentOutOfRangeException
startIndex darf nicht länger als die Länge der Zeichenfolge sein.
Parametername: startIndex
System.String InternalSubStringWithChecks(Int32, Int32, Boolean) @ System.String
mscorlib
   bei System.String.InternalSubStringWithChecks(Int32 startIndex, Int32 length, Boolean fAlwaysCopy)
   bei GitCommands.GitModule.FindGitSuperprojectPath(String& submoduleName, String& submodulePath)
   bei GitCommands.GitModule.get_SuperprojectModule()
   bei GitUI.CommandsDialogs.FormBrowse.UpdateSubmodulesList()
startIndex
-2146233086
```
```
$ git --version
git version 2.5.2.windows.2
```

**Comment by vbjay** [comment]
:+1:

**Comment by mindw** [comment]
:+1:

**Comment by vcpp** [comment]
Duplicates #2929, but this issue has better description.
I've got the same stacktrace when started using worktrees and fixed it in #3125.
